Modules/Core/SpatialObjects/include/itkTubeSpatialObjectPoint.hxx

```diff
@@ -46,6 +46,14 @@ double
 TubeSpatialObjectPoint< TPointDimension >
 ::GetRadiusInWorldSpace() const
 {
+  if( m_SpatialObject == nullptr )
+    {
+    ExceptionObject e(__FILE__);
+    e.SetLocation( "SpatialObjectPoint:GetRadiusInWorldSpace" );
```

Comment:
Don't the [macros](https://github.com/InsightSoftwareConsortium/ITK/blob/95f095a7f7e366cc2fd7dc87acb4c50a71b85568/Modules/Core/Common/include/itkMacro.h#L432-L462) do a better job at providing information about location? E.g. `itkExceptionMacro("The SpatialObject must be set prior to calling.")`

Reply:
Changed all ExceptionObject uses to itkExceptionMacro() uses.